client/src/app/modules/error-handler/http-request-interceptor.ts

```diff
@@ -39,6 +34,16 @@ export class HttpRequestInterceptor implements HttpInterceptor {
 
     req = req.clone(httpOptions);
 
-    return next.handle(req);
+    return next.handle(req).pipe(
+      retry(2),
```

Comment:
When I get it right, this is retrying any kind of failed request two more times automatically. This might also happen with partially successful requests and might lead to creation of even more broken parts or cause other issues on top. I would not do a retry automatically.

Reply:
You'e right, it"s better to leave it to the component to retry requests, so I removed it here. Will think about in which components this would make sense to tackle (incl. a `retryWhen` to not fire constantly).